config/rewrites.d/developer.rackspace.com.json

```diff
@@ -350,7 +350,28 @@
         {
           "description": "Redirect all Rackspace KaaS docs to /latest",
           "from": "^\\/docs\\/rkaas\\/1.0.x\\/(.*)",
-          "to": "/docs/rkaas/latest/$1",
+          "to": "/docs/rkaas/latest/external/$1",
+          "rewrite": false,
+          "status": 301
+        },
+        {
+          "description": "Redirect Rackspace KaaS User Guide",
+          "from": "^\\/docs\\/rkaas\\/latest\\/rkaas-userguide-external\\/?(.*)",
+          "to": "/docs/rkaas/latest/external/rkaas-userguide/$1",
+          "rewrite": false,
+          "status": 301
+        },
+        {
+          "description": "Redirect Rackspace KaaS Release Notes",
+          "from": "^\\/docs\\/rkaas\\/latest\\/rkaas-releasenotes-external/\\/?(.*)",
```

Comment:
Should the forward slash after "external" be there?

Reply:
yes! good catch!